wcfsetup/install/files/style/bbcode/code.scss

```diff
@@ -151,83 +151,168 @@
 	position: static;
 }
 
-/* PrismJS 1.29.0
-https://prismjs.com/download.html#themes=prism */
-/**
- * prism.js default theme for JavaScript, CSS and HTML
- * Based on dabblet (http://dabblet.com)
- * @author Lea Verou
- */
-
-.prism-token.prism-comment,
-.prism-token.prism-prolog,
-.prism-token.prism-doctype,
-.prism-token.prism-cdata {
-	color: slategray;
-}
+html:not([data-color-scheme="dark"]) {
```

Comment:
```suggestion
html[data-color-scheme="light"] {
```

Reply:
> Do you really need this at all?

The two Prism styles are completely independent stylesheets obtained from an external source. I don't want any light mode styling to leak into the dark mode styling, because that will likely result in non-sense.